api/k8s/v1/k8s.proto

```diff
@@ -288,6 +288,10 @@ message Pod {
 
   map<string, string> labels = 9;
   map<string, string> annotations = 10;
+
+  // human friendly, based off
+  // https://github.com/lyft/lyftkube/blob/1394f8f8e17d9f3472a0387c901bdb506dde083a/lib/printers/time_format.go#L11
+  string age = 11;
```

Comment:
So is the plan to have `start time` and `age` as separate fields in Pod? Feels redundant to have both

Reply:
I can remove `start time` for now.

CreationTimestamp comment in types.go: 
	// CreationTimestamp is a timestamp representing the server time when this object was
	// created. It is not guaranteed to be set in happens-before order across separate operations.
	// Clients may not set this value. It is represented in RFC3339 form and is in UTC.
	//
	// Populated by the system.
	// Read-only.
	// Null for lists.
	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata

StartTime comment from types.go:
	// RFC 3339 date and time at which the object was acknowledged by the Kubelet.
	// This is before the Kubelet pulled the container image(s) for the pod.